docs/how-to-guides/pro-development.md

```diff
@@ -0,0 +1,44 @@
+# develop on the pro platform
+
+Here is how to develop for the pro platform using docker.
+
+It suppose [you already have setup docker for dev](../introduction/dev-environment-quick-start-guide.md).
+
+You should have two kind of shell:
+- the shell for openfoodfacts
+- the shell for openfoodfacts-pro, this is a shell where you have source the `setenv-pro.sh`,
+  that is you run `. setenv-pro.sh`.
+  Your prompt, should now contains a `(pro)` to recall you you are in pro environment.
+  (this simply sets some environment variables that will overides the one in .env)
+
+To develop, on pro plateform, you can then us a shell for openfoodfacts-pro and simply do a `make dev` and everything as usual.
+
+If you need to work on product import/export, or interacting with public platform,
+you have to start postgres and the minion on off side.
+That is, in a *non pro* shell, run `docker-compose up postgres minion`.
+
+Note that the setup does not currently support running the http server for both public and pro platform at the same time.
```

Comment:
yes but `openfoodfacts.localhost` and `openfoodfacts.pro.localhost` have to be different instance of product opener.
Those instances are not on the same network.
If we want to have nginx from one project serving both, it will require a specific network + more configuration. 
I think this is out of scope for:
- on preprod / prod the host nginx will do the job
- on dev not many people have engouh RAM to run two instances of productopener, and you can start / stop the one you need, to check everything is ok (import / export does not need the public instance to run).